forum/users/forms.py

```diff
@@ -0,0 +1,14 @@
+from django import forms
+from .models import Role
+from django.contrib.auth.forms import AuthenticationForm
+
+class UserLoginForm(AuthenticationForm):
+    username = forms.EmailField(label="Email")
+    password = forms.CharField(widget=forms.PasswordInput, label="Password")
```

Comment:
If you're looking for an extra layer of security for the password field, you could add `autocomplete="off"` in the widget to prevent password suggestions in browsers